build.csx

```diff
@@ -11,66 +11,57 @@ var solution = "./" + solutionName + ".sln";
 var packagesDirectory = Path.GetFullPath("packages");
 var repoUrl = "https://github.com/FakeItEasy/FakeItEasy";
 var coverityProjectUrl = "https://scan.coverity.com/builds?project=FakeItEasy%2FFakeItEasy";
-var versionAssembly = "./src/FakeItEasy/bin/Release/FakeItEasy.dll";
+var versionAssembly = "./src/FakeItEasy/bin/Release/net40/FakeItEasy.dll";
 var nuspecs = Directory.GetFiles("./src", "*.nuspec");
-var gitlinks = Directory.GetDirectories("./src").Where(p => !p.EndsWith(".Shared")).Select(Path.GetFileName);
-
-var unitTests = new []
-{
-    "tests/FakeItEasy.Tests/bin/Release/FakeItEasy.Tests.dll",
-    "tests/FakeItEasy.Analyzer.CSharp.Tests/bin/Release/FakeItEasy.Analyzer.CSharp.Tests.dll",
-    "tests/FakeItEasy.Analyzer.VisualBasic.Tests/bin/Release/FakeItEasy.Analyzer.VisualBasic.Tests.dll"
-};
-
-var netstdUnitTestDirectories = new []
-{
-    "tests/FakeItEasy.Tests.netstd"
-};
-
-var integrationTests = new []
-{
-    "tests/FakeItEasy.IntegrationTests/bin/Release/FakeItEasy.IntegrationTests.dll",
-    "tests/FakeItEasy.IntegrationTests.VB/bin/Release/FakeItEasy.IntegrationTests.VB.dll"
-};
-
-var netstdIntegrationTestDirectories = new []
-{
-    "tests/FakeItEasy.IntegrationTests.netstd"
-};
-
-var specs = new []
-{
-    "tests/FakeItEasy.Specs/bin/Release/FakeItEasy.Specs.dll"
-};
-
-var netstdSpecDirectories = new []
+var pdbs = new []
 {
-    "tests/FakeItEasy.Specs.netstd"
+    "src/FakeItEasy/bin/Release/net40/FakeItEasy.pdb",
+    "src/FakeItEasy/bin/Release/netstandard1.6/FakeItEasy.pdb",
+    "src/FakeItEasy.Analyzer/bin/Release/netstandard1.1/FakeItEasy.Analyzer.Csharp.pdb",
+    "src/FakeItEasy.Analyzer/bin/Release/netstandard1.1/FakeItEasy.Analyzer.VisualBasic.pdb"
 };

```

Comment:
This is fine, but we could consider just adding using `src/*/bin/Release/*/*.pdb`.